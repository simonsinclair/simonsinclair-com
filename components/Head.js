import NextHead from 'next/head';

const Head = () => (
  <NextHead>
    <meta charSet="UTF-8" />
    <title>Simon Sinclair &mdash; Home</title>
    <meta name="description" content="" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
  </NextHead>
);

export default Head;
