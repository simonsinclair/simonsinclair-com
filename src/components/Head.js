import NextHead from 'next/head';

const Head = () => (
  <NextHead>
    <meta charSet="UTF-8" />
    <title>Simon Sinclair &mdash; Home</title>
    <meta name="description" content="Simon Sinclair is a multidisciplinary UX Designer living in London, UK." />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
  </NextHead>
);

export default Head;
