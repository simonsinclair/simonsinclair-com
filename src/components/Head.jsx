import NextHead from 'next/head';
import PropTypes from 'prop-types';

const Head = ({ children }) => (
  <NextHead>
    <meta charSet="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    { children }
  </NextHead>
);

Head.propTypes = {
  children: PropTypes.arrayOf(PropTypes.element).isRequired,
};

export default Head;
