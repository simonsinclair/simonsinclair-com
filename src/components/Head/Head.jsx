import NextHead from 'next/head';
import PropTypes from 'prop-types';

const Head = ({ children }) => (
  <NextHead>
    <meta charSet="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    { children }
  </NextHead>
);

Head.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Head;
