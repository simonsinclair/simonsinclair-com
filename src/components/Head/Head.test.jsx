import { render } from '@testing-library/react';

import Head from './Head';

describe('<Head /> is a component that', () => {
  it('renders and matches its snapshot', () => {
    const { container } = render(<Head><title>Dummy Title</title></Head>);
    expect(container).toMatchSnapshot();
  });
});
