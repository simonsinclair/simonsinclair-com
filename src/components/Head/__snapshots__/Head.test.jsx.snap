// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Head /> is a component that renders and matches its snapshot 1`] = `<div />`;
